rgd/geodata/tests/conftest.py

```diff
@@ -9,8 +9,13 @@
 
 
 @pytest.fixture
-def api_client() -> APIClient:
-    return APIClient()
+def api_client(user) -> APIClient:
+    client = APIClient()
+    user.is_superuser = True
+    user.is_staff = True
+    user.is_active = True
```

Comment:
You're not saving the `user` here, which might be fragile. I think it'd be better to use the `user_factory` fixture, and pass these attributes as parameters when instantiating a new user with the factory.

Reply:
Resolved in bd4c14e7d8661e0da7734edaf10b51610b7e6cbf